config.env.template

```diff
@@ -16,3 +16,10 @@ oapen_ftp_pw=
 # University of Cambridge Apollo DSpace 7 test server credentials
 cam_ds7_user=
 cam_ds7_pw=
+
+# Crossref user credentials
+# Separate credentials must be provided for each publisher whose works are to be submitted.
+# Each set of credentials must be given in the format below, i.e. with the
+# Thoth publisher ID suffixed (omitting the square brackets).
+crossref_user_[publisher_id]=
```

Comment:
This won't work as-is on GitHub Actions as the publisher ID contains hyphens, which are not allowed in repository secret names.

Reply:
Fixed the above under #16 - please also check this (small) PR as part of this review.